tests/python/relay/test_conv2d_nchw_texture.py

```diff
@@ -392,3 +392,81 @@ def test_conv2d_yolov3_v2_nchw_3c():
     }
 
     build_run_compare(mod, params, {"data": input_shape}, dtype, target)
+
+
+def test_conv2d_vgg16_winograd():
```

Comment:
can we verify in the test if winograd was selected? I.e. take json after compilation and look for winograd in the functin names?